app/build.gradle

```diff
@@ -88,7 +88,7 @@ dependencies {
     implementation "androidx.recyclerview:recyclerview:$recyclerview_version"
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.exifinterface:exifinterface:1.0.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha4'
+    implementation "androidx.constraintlayout:constraintlayout:2.0.0-alpha5"
```

Comment:
I have to add this so that there is not any lint error, @sashei is that fine? would not like to merge the PR without approval on this change (after the review...) Thanks!